rotkehlchen/chain/arbitrum_one/modules/thegraph/balances.py

```diff
@@ -22,3 +30,16 @@ def __init__(
             native_asset=A_GRT_ARB,
             staking_contract=CONTRACT_STAKING,
         )
+
+    def query_balances(self) -> BalancesSheetType:
```

Comment:
What are you doing here? This is identical to the ethereum one. Word for word. Didn't I tell you to abstract it? That's not how abstractions work. Just move this to the superclass please with the location being decided depending on the class